pandas/tests/extension/test_categorical.py

```diff
@@ -41,14 +41,19 @@ def dtype():
     return CategoricalDtype()
 
 
-@pytest.fixture
```

Comment:
pls don't change the fixture setup; these are very explict and match every other extension type.

Reply:
ok, will change them all in a dedicated pass (as discussed at the sprint)

Reply:
right if you change them *all* prob ok, but separate pass